src/utils/__tests__/cosmosdb_model.test.ts

```diff
@@ -288,3 +284,66 @@ describe("find", () => {
     }
   });
 });
+
+const anotherTest = "another-test";
+describe("patch", () => {
+  it("GIVEN an existing document WHEN patch a value THEN return a task either containing the updated document", async () => {
+    containerMock.item.mockReturnValueOnce({ patch: patchMock });
+    patchMock.mockImplementationOnce(() =>
+      Promise.resolve({
+        resource: {
+          ...aDocument,
+          ...someMetadata,
+          test: anotherTest
+        }
+      })
+    );
+    const model = new MyModel(container);
+    const result = await model.patch([testId], { test: anotherTest })();
+
+    expect(patchMock).toBeCalledWith(
+      [{ op: "add", path: "/test", value: anotherTest }],
+      undefined
+    );
+    expect(E.isRight(result)).toBeTruthy();
+    if (E.isRight(result)) {
+      expect(result.right.test).toEqual(anotherTest);
+    }
+  });
```

Comment:
Since we are mocking the patch result returning a promise with `test: anotherTest` , what is the value of checking `result`?

Reply:
you are right, removed!